app/controllers/verify/sessions_controller.rb

```diff
@@ -55,10 +55,25 @@ def idv_profile_form
       @_idv_profile_form ||= Idv::ProfileForm.new((idv_session.params || {}), current_user)
     end
 
+    # rubocop:disable MethodLength
```

Comment:
looks like sometimes we are disabling these in `.rubocop.yml` and sometime inline. I probably prefer former but either way is fine 🐱